### PR TITLE
Added condition to skip terminated containers

### PR DESCRIPTION
**Pull request by hpandeycodeit**
Fixes Issue #13263

### CODE REVIEW

File: prow/cmd/deck/main.go [hunk 1]
```diff
@@ -1104,8 +1104,11 @@ func handleLog(lc logClient) http.HandlerFunc {
 				// for a Pod or ProwJob that doesn't exit, it's not
 				// something an administrator wants to see in logs.
 				logger.Info(msg)
+			} else if strings.Contains(err.Error(), "terminated") {
```
Comment (stevekuznetsov) [inline prow/cmd/deck/main.go]: Could we add this to the list of things we check in the above condition instead?

Comment (hpandeycodeit) [reply prow/cmd/deck/main.go]: yes. Not sure why I didn't add in the original condition initially :)

**Comment by stevekuznetsov** [comment]
/approve
:tada:

**Comment by hpandeycodeit** [comment]
> /approve
> 🎉

yay!! 🎉

**Pull request merged**